### ISSUE TITLE
virtio network names have changed between systemd v225 and v229

### ISSUE THREAD

**Issue by mischief** [coreos/bugs#1168]
in coreos running under qemu with systemd v225, the first nic is called eth0, but with systemd v229 the nic is called ens3. this behavior might have changed on gce too.

coreos should probably ship a udev or networkd rule to keep compatibility.

cc @marineam @crawford

**Comment by mischief** [comment]
this worked for me in qemu -

```ini
[Match]
Driver=virtio_net

[Link]
NamePolicy=onboard kernel
```

but i wonder, can we apply this only on an upgrade for compat and not for new installs?

**Comment by mischief** [comment]
should be fixed by coreos/update_engine#101

**Comment by mischief** [comment]
fixed by coreos/update_engine#101
